src/pretix/control/forms/orders.py

```diff
@@ -118,3 +118,9 @@ class OrderContactForm(forms.ModelForm):
     class Meta:
         model = Order
         fields = ['email']
+
+
+class OrderLocaleForm(forms.ModelForm):
+    class Meta:
+        model = Order
+        fields = ['locale']
```

Comment:
@raphaelm where is `event.settings.locales`? I don't seem to find it or understand what that is